src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -15,14 +15,21 @@ public class StaticNullChecking : CompilingTestBase
         [Fact]
         public void TestAll()
```

Comment:
The test has been useful to help catch cases of cycles in lazy evaluation (particularly calculating `TypeSymbol.IsReference`). The test is marked with a `// PROTOTYPE` to remove in the future.